source/manuals/programming-languages/html.html.md.erb

```diff
@@ -94,6 +69,31 @@ Examples include author information, related links and related content.
 
 `<footer role="contentinfo">` should be used for the footer of the site.
 
+### Region landmarks
+
+Region landmarks can be used when you need to create a new type of landmark.
```

Comment:
I don't really understand what 'new type of landmark' means here.

Cribbing a little from https://www.scottohara.me/blog/2018/03/03/landmarks.html, what about something like:

> Use region landmarks to surface important areas of a page where it's not appropriate to use [any of the other types of landmarks](https://www.scottohara.me/blog/2018/03/03/landmarks.html).

Though that does introduce some repetition with the second bullet point below? 🤔

Reply:
I think what I was trying to say with 'new type of landmark' is that 'region' on its own doesn't really mean anything, you give the meaning through the accessible name. Because you're in control of that meaning, my brain saw that as you creating a new type of landmark but, with a bit of hindsight, I agree that that way of describing it is confusing as it's not like you're creating a new role.

I also think it duplicating bulleted content below can be easily fixed by just deleting the duplicate and making the bullets a sentence. Will push a commit to do include this change and do that.